controllers/twophase/types.go

```diff
@@ -92,15 +92,50 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 		status.Experiment.Phase = v1alpha1.ExperimentPhaseFinished
 		status.FailedMessage = emptyString
-	} else if chaos.IsPaused() {
+	} else if chaos.GetPause() != "" {
 		if status.Experiment.Phase == v1alpha1.ExperimentPhaseRunning {
 			r.Log.Info("Pausing")
 
-			err = r.Recover(ctx, req, chaos)
-			if err != nil {
-				r.Log.Error(err, "failed to pause chaos")
-				updateFailedMessage(ctx, r, chaos, err.Error())
-				return ctrl.Result{Requeue: true}, err
+			if chaos.GetPause() == "true" {
```

Comment:
`If chaos.GetPause() == "false“`?

Reply:
So we still have the original pause&unpause, that if we set pause to `true`, we use the original method. Otherwise when we set pause to some duration, we use the new one.